doc/page-cmakefunctions.h

```diff
@@ -76,7 +80,25 @@ facelift_add_library(<TargetName>
 \endcode
 
 This function adds a library with the name \e TargetName from the specified source and header files. The parameter descriptions can be found below.
-
+In the parameter HEADERS_NO_MOC_FILE you can set the file with a list of the files which sould be excluded from processing by moc.
+This set should have the name HEADERS_NO_MOC_FROM_FILE. E.g in we create the file ~/nomoc.cmake and fill it like this:
```

Comment:
HEADERS_NO_MOC is the only parameter which needs to be documented

Reply:
HEADERS_NO_MOC_FROM_FILE needed, please see  comment above.